templates/rails/files/.platform.app.yaml

```diff
@@ -0,0 +1,35 @@
+name: "app"
```

Comment:
Please, don't forget to put details on this file, e.g.:
https://github.com/platformsh/template-drupal8/blob/master/.platform.app.yaml